emails/tests/models_tests.py

```diff
@@ -1293,6 +1309,7 @@ def test_clear_storage_with_update_fields(self) -> None:
         domain_address = DomainAddress.objects.create(
             user=self.storageless_user, address="no-storage"
         )
+        assert domain_address.used_on == None
```

Comment:
There is a later `asserts` in this test that need to change as well, looks like like 1333. The `domain_address.used_on` should be cleared after running `domain_address.save(update_fields={"last_used_at"})`